include/class.filter.php

```diff
@@ -369,6 +369,7 @@ function delete() {
         $sql='DELETE FROM '.FILTER_TABLE.' WHERE id='.db_input($id).' LIMIT 1';
         if(db_query($sql) && ($num=db_affected_rows())) {
             db_query('DELETE FROM '.FILTER_RULE_TABLE.' WHERE filter_id='.db_input($id));
+            db_query('DELETE FROM '.FILTER_ACTION_TABLE.' WHERE filter_id='.db_input($id));
```

Comment:
This works, however, as a result of some of our latest commits, the filter_id on filter actions isn't being set. I'll look into it